admin/section-column-resize.php

```diff
@@ -40,9 +43,11 @@
 
 		$block_sizes[] = (int) $block_columns;
 
-		$block_increment++;
+		$block_increment ++;
 	}
-endif;
-if ( (int) $section_blocks > 1 && (int) $section_blocks < 4 ) : ?>
+}
+
+if ( $section_blocks > 1 && $section_blocks < 4 ) : ?>
```

Comment:
Curious why some conditional checks with $section_blocks are type casted as `(int)` while others aren't, this line is an example of where the type casting was removed